README.md

```diff
@@ -304,6 +304,10 @@ Once the user clicks on "Allow" in the login dialog, the Nextcloud Files App wil
 - [Nextcloud Deck app](https://github.com/stefan-niedermann/nextcloud-deck/)
   - [API](https://github.com/stefan-niedermann/nextcloud-deck/blob/master/app/src/main/java/it/niedermann/nextcloud/deck/api/DeckAPI.java)
   - [Login](https://github.com/stefan-niedermann/nextcloud-deck/blob/master/app/src/main/java/it/niedermann/nextcloud/deck/ui/ImportAccountActivity.java#L77)
+- [Nextcloud Bookmarks app](https://gitlab.com/bisada/OCBookmarks)
+  - [API](https://nextcloud-bookmarks.readthedocs.io/en/latest/)
```

Comment:
With `API` is not the server part of the API meant but the part of the client app that connects to the REST endpoints - in your case this is here: `https://gitlab.com/bisada/OCBookmarks/-/blob/master/app/src/main/java/org/schabi/ocbookmarks/REST/OCBookmarksRestConnector.java`